docker/Dockerfile.mediamtx

```diff
@@ -18,14 +39,8 @@ RUN	mkdir -p /var/log/ \
 
 COPY --chmod=0755	mediamtx-metrics.bash	/opt/mediamtx-metrics.bash
 
-ENV	MEDIAMTX_VERSION="1.9.3"
-
-ADD	"https://github.com/bluenviron/mediamtx/releases/download/v${MEDIAMTX_VERSION}/mediamtx_v${MEDIAMTX_VERSION}_linux_amd64.tar.gz"	/opt/mediamtx/mediamtx.tar.gz
-
-RUN	tar xzf /opt/mediamtx/mediamtx.tar.gz -C /opt/mediamtx/ \
-	&& mkdir -p /usr/local/bin /etc/mediamtx/ \
-	&& mv /opt/mediamtx/mediamtx /usr/local/bin/mediamtx \
-	&& mv /opt/mediamtx/mediamtx.yml /etc/mediamtx/mediamtx.yml \
-	&& rm -rf /opt/mediamtx/
+# Copy artifacts from the builder stage
```

Comment:
since this is temporary, should we put the removed snippet as comments instead of deleting completely?

Reply:
and update the `MEDIAMTX_VERSION` with `v1.11.2-livepeer-1` as we track that version field for alerts in deployments/version mismatches.

Reply:
I'm ok with still deleting, we have the git history to go back to. not really a big fan of commented out code

Reply:
Re-introduced the MEDIAMTX_VERSION env var in https://github.com/livepeer/go-livepeer/pull/3353/commits/d6d05ed0ecebecf2e634eb492ba926cd2e3a192e

> since this is temporary, should we put the removed snippet as comments instead of deleting completely?

I am not sure exactly how temporary it would be ... depends on how soon our changes get upstreamed, and whether there is anything else we need to change and maintain for ourselves. Could be a while. Like Max said, we have the git history to fall back to.